hibernate-search-elasticsearch/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,284 @@
+            _clearForm();
+        }
+
+        function _clearForm() {
+            $scope.form.author = {
+                id: -1,
+                firstName: '',
+                lastName: ''
+            };
+            $scope.form.book = {
+                id: -1,
+                title: '',
+                authorId: ''
+            };
+        }
+
+        function _error(response) {
+            alert(response.data.message || response.statusText);
+            _clearForm();
+        }
+      }]);
+    </script>
+</head>
+<body ng-app="LibraryManagement" ng-controller="LibraryManagementController">
+    <div class="ui fixed inverted menu">
+        <div class="ui container">
+            <a class="header item" href="https://quarkus.io/">
+                <img class="logo" src="https://quarkus.io/assets/images/quarkus_logo_horizontal_rgb_reverse.svg" alt="Quarkus logo" style="width:200px;">
+            </a>
+            <span class="header item">
+                Hibernate Search + Elasticsearch quickstart
+            </span>
+        </div>
+    </div>
+    <div class="presentation">
+        <div class="ui main container">
+            <p>This Quarkus application demonstrates how to implement a CRUD endpoint to manage a library using Hibernate ORM with Panache and Hibernate Search.</p>
+            <p>Entities are automatically indexed in an Elasticsearch cluster.</p>
+            <div class="ui tablet stackable three steps">
+                <div class="step">
+                    <i class="list icon"></i>
+                    <div class="content">
+                        <div class="title">RESTEasy</div>
+                        <div class="description">JAX-RS endpoints</div>
+                    </div>
+                </div>
+                <div class="step">
+                    <i class="random icon"></i>
+                    <div class="content">
+                        <div class="title">Hibernate ORM<br />Hibernate Search</div>
+                        <div class="description">Store in the database and index automatically</div>
+                    </div>
+                </div>
+                <div class="step">
+                    <i class="database icon"></i>
+                    <div class="content">
+                        <div class="title">PostgreSQL<br />Elasticsearch</div>
+                        <div class="description">Relational mapping and full text search</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="ui container search">
+        <form class="ui form" ng-submit="refresh()">
+            <div class="fluid big left ui action input">
+                <input type="text" placeholder="Search for an author name or a book title. Try searching for aus*, owen or mystery." ng-model="pattern">
+                <button class="ui blue right labeled icon button" ng-click="refresh()"><i class="search icon"></i> Search</button>
```

Comment:
It would be great to also grey out books that do not match the search pattern. It would only require a second search query, and I think would make the demo even more appealing.

Reply:
Also, I think removing the button and updating the list after a slight delay without typing (~200ms) would feel better. It's not like we have to care about overloading the server :) Not sure how easy that would be, though.

Reply:
Yeah, so we need to be very careful about what we do. The demo as it is is already on the complex side when trying to follow the guide. Keep in mind that's this demo is really a companion for the guide.

We could improve things but it will come with the cost of added complexity for the guide, which wouldn't be a good idea IMHO.